providers/stores/azblobHarvestStore.js

```diff
@@ -46,34 +46,40 @@ class AzHarvestBlobStore extends AbstractAzBlobStore {
    * @param {EntityCoordinates} coordinates - The component revision to report on
    * @returns An object with a property for each tool and tool version
    */
-  getAll(coordinates) {
-    const name = this._toStoragePathFromCoordinates(coordinates)
+  async getAll(coordinates) {
     // Note that here we are assuming the number of blobs will be small-ish (<10) and
     // a) all fit in memory reasonably, and
     // b) fit in one list call (i.e., <5000)
-    const list = new Promise((resolve, reject) =>
+    const files = await this._getAllFiles(coordinates)
+    return await this._getContent(files)
+  }
+
+  _getAllFiles(coordinates) {
```

Comment:
Suggesting name change here and in fileHarvestStore to: `_getListOfAllFiles(coordinates)`